tests/end2end/Migrations-test.js

```diff
@@ -0,0 +1,74 @@
+import {E2EApp} from './helpers/E2EApp';
+import {E2EMeetingSeries} from './helpers/E2EMeetingSeries';
+import {E2EMinutes} from './helpers/E2EMinutes';
+import {E2ETopics} from './helpers/E2ETopics';
+import {E2EGlobal} from './helpers/E2EGlobal';
+
+describe.skip('Migrations', function () {
```

Comment:
Is this `skip` supposed to to be in here?

Reply:
Yes. I introduced this "test" to figure out which migration caused the bug by inserting some data, migrating back to an old version and then migrating step by step one version up. After each migration I checked the amount of items of each topic and threw an error if one item is missing. Later I used this test again to verify if MigrationV21 did its job by verifying the amount of items only after the last migration.

Since this test is really slow and takes static migration version numbers (from 17 to 21) I think it is not very helpful when we run it continously. Indeed, it might be helpful in the future if we would have similar issue with our migrations.

Reply:
Ok